src/i18n/es.js

```diff
@@ -0,0 +1,8 @@
+export const messages_es = {
+  alignment: 'Alineación',
+  big: 'Grande',
+  regular: 'Normal',
+  size: 'Tamaño',
+  small: 'Pequeño',
+  social_networks: 'Redes sociales',
```

Comment:
What do you think about using the same standards as in our other projects?

```suggestion
  alignment: `Alineación`,
  big: `Grande`,
  regular: `Normal`,
  size: `Tamaño`,
  small: `Pequeño`,
  social_networks: `Redes sociales`,
```

Reply:
I think it is unnecessary this suggest, I like use the `` when need escape the text with variables because the transpilation will try interpolate the string without reason.

Reply:
In the abstract, I agree with you. But, in other projects like [WebApp](https://github.com/FromDoppler/doppler-webapp/blob/master/src/i18n/es.js) and [Editors Webapp](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/components/i18n/es.tsx), [we have chosen` symbol as standard](https://github.com/FromDoppler/doppler-webapp/blob/master/src/i18n/README.md):

> In the result object, define all string properties using [ES6 Template Strings](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings), in order to support line breaks. 

The rationale is here in PR https://github.com/FromDoppler/doppler-webapp/pull/173

Reply:
Resolved